ui/js/graphs.js

```diff
@@ -669,14 +669,26 @@ perf.controller('GraphsCtrl', [
                 ThRepositoryModel.get_list().then(function(response) {
 
                     $scope.projects = response.data;
-                    $scope.addTestData = function() {
+                    $scope.addRelatedPlatform = function(seriesSignature) {
+                        $scope.addTestData("addRelatedPlatform", seriesSignature);
```

Comment:
Hi Will,
I'm haven't fully understand about how to apply the enumed parameter in here, but I change the code by my own understanding. :)